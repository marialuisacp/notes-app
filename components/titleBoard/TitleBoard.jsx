import React from 'react';
import StyledTitleBoard from './styles/TitleBoard.styles';

const TitleBoard = () => (
  <StyledTitleBoard>
    Onboarding - Maria Lu√≠sa
  </StyledTitleBoard>
);

export default TitleBoard;