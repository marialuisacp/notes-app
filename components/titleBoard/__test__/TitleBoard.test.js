import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import TitleBoard from '../TitleBoard';

test('should render component TitleBoard', () => {
  const { fragment } = render(<TitleBoard></TitleBoard>);
  expect(fragment).toMatchSnapshot();
});
