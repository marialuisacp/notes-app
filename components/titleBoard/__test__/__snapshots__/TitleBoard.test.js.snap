// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component TitleBoard 1`] = `undefined`;
